daemon/cmd/daemon_main.go

```diff
@@ -852,6 +852,11 @@ func initEnv(cmd *cobra.Command) {
 		}
 	}
 
+	// This check is here instead of in .Populate above to avoid an import loop.
```

Comment:
Instead of in where? I'm on the edge of my seat..

Reply:
I guess the Populate call was quite a ways away. It wasn't a sentence ending full-stop :P

Reply:
Unexpected parser error ;)